examples/todos/package.json

```diff
@@ -5,7 +5,6 @@
   "scripts": {},
   "license": "MIT",
   "dependencies": {
-    "@elastic/synthetics": "*",
-    "playwright-core": "=1.6.2"
+    "@elastic/synthetics": "file:../../"
```

Comment:
will change once we merge and release a new version where types would become available.